Getting lots of complaints about `std::chrono` on Windows.
```
 D:\a\alpaka\alpaka\include\alpaka\test\MeasureKernelRunTime.hpp(20,41): error C2039: 'high_resolution_clock': is not a member of 'std::chrono' [D:\a\alpaka\alpaka\build\test\integ\axpy\axpyTest.vcxproj]
  (compiling source file '../../../../test/integ/axpy/src/axpy.cpp')
      C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.43.34808\include\__msvc_chrono.hpp(286,11):
      see declaration of 'std::chrono'
```
Not sure to what extent this is related to this PR. Any ideas how to proceed, @psychocoderHPC or @SimeonEhrig ?

--- a/include/alpaka/test/MeasureKernelRunTime.hpp
+++ b/include/alpaka/test/MeasureKernelRunTime.hpp
@@ -7,6 +7,7 @@
 #include "alpaka/alpaka.hpp"
 #include "alpaka/core/DemangleTypeNames.hpp"
 
+#include <chrono>
 #include <type_traits>
 #include <utility>
 

